x-pack/plugins/canvas/public/components/asset_manager/__stories__/__snapshots__/asset.stories.storyshot

```diff
@@ -355,181 +355,3 @@ exports[`Storyshots components/Assets/Asset marker 1`] = `
-              <div
-                aria-hidden="true"
-                className="euiButtonIcon__icon"
-                data-euiicon-type="vector"
-                size="m"
-              />
-            </button>
-          </span>
-        </div>
-        <div
-          className="euiFlexItem euiFlexItem--flexGrowZero asset-download"
-        >
-          <span
-            className="euiToolTipAnchor"
-            onKeyUp={[Function]}
-            onMouseOut={[Function]}
-            onMouseOver={[Function]}
-          >
-            <div
-              className="canvasDownload"
-              onClick={[Function]}
-              onKeyPress={[Function]}
-              role="button"
-              tabIndex={0}
-            >
-              <button
-                aria-label="Download"
-                className="euiButtonIcon euiButtonIcon--primary"
-                type="button"
-              >
-                <div
-                  aria-hidden="true"
-                  className="euiButtonIcon__icon"
-                  data-euiicon-type="sortDown"
-                  size="m"
-                />
-              </button>
-            </div>
-          </span>
-        </div>
-        <div
-          className="euiFlexItem euiFlexItem--flexGrowZero"
-        >
-          <span
-            className="euiToolTipAnchor"
-            onKeyUp={[Function]}
-            onMouseOut={[Function]}
-            onMouseOver={[Function]}
-          >
-            <div
-              className="canvasClipboard"
-              onClick={[Function]}
-              onKeyPress={[Function]}
-              role="button"
-              tabIndex={0}
-            >
-              <button
-                aria-label="Copy id to clipboard"
-                className="euiButtonIcon euiButtonIcon--primary"
-                type="button"
-              >
-                <div
-                  aria-hidden="true"
-                  className="euiButtonIcon__icon"
-                  data-euiicon-type="copyClipboard"
-                  size="m"
-                />
-              </button>
-            </div>
-          </span>
-        </div>
-        <div
-          className="euiFlexItem euiFlexItem--flexGrowZero"
-        >
-          <span
-            className="euiToolTipAnchor"
-            onKeyUp={[Function]}
-            onMouseOut={[Function]}
-            onMouseOver={[Function]}
-          >
-            <button
-              aria-label="Delete"
-              className="euiButtonIcon euiButtonIcon--danger"
-              onBlur={[Function]}
-              onClick={[Function]}
-              onFocus={[Function]}
-              type="button"
-            >
-              <div
-                aria-hidden="true"
-                className="euiButtonIcon__icon"
-                data-euiicon-type="trash"
-                size="m"
-              />
-            </button>
-          </span>
-        </div>
-      </div>
-    </div>
-  </div>
-</div>
-`;
```

Comment:
Can we add this back? Might need to be generated by running with jest in our canvas directory

Reply:
Talked with Poff on slack, turns out we don't need this afterall